test/suites/network/multi-nic.robot

```diff
@@ -21,7 +21,7 @@ ${USHIFT_HOST_IP2}      ${EMPTY}
 ${NIC1_NAME}            ${EMPTY}
 ${NIC2_NAME}            ${EMPTY}
 ${NICS_COUNT}           2
-${NMCLI_CMD}            nmcli -f name,type connection | awk '$2 == "ethernet" {print $1}' | sort
+${NMCLI_CMD}            nmcli connection | grep ethernet
```

Comment:
Why did you change the command?

Reply:
Because on QE env I can not get the list of interfaces using that command. On QE env connection names have 3 strings, so `print $1` on `awk` command won't work:
```
$ nmcli -f name,type connection
NAME                TYPE
Wired connection 1  ethernet
Wired connection 2  ethernet
Wired connection 3  ethernet
cni-podman0         bridge
lo                  loopback
enp1s0              ethernet

Reply:
thanks, that also works
```suggestion
${NMCLI_CMD}            nmcli -f name,type connection | awk '$NF == "ethernet" { $NF=""; print $0 }' | sort
```